### DISCUSSION TITLE
LAB 02.03 - Task 1: Extract data

### DISCUSSION THREAD

**Discussion by Andresfga** [rramosp/ai4eng.v1#74] [02 - PYTHON]
Buenas noches, tengo un problema al solucionar esta tarea ya que me piden los indices de los price y de los margin que cumplan la condición dada en en enunciado, en si con el código calculo lo que necesito con la funcion index, pero la lista que me arroja contiene al final las palabras , dtype='int64', name='itemid' y la parte inicial Int64Index y no se como solucionar esto para que solo me salga el valor del indice, ojala puedan ayudarme, muchas gracias.

![Captura](https://user-images.githubusercontent.com/95330160/144169638-2decbd99-e91e-4aaa-85bd-0d69669e443b.PNG)
![Captura2](https://user-images.githubusercontent.com/95330160/144169709-8b0da56e-7fe9-40f2-a55c-3b0626258fb9.PNG)

**Comment by rramosp** [comment]
lo que te está devolviendo es un índice, simplemente tienes que convertirlo
a lista

result = list(result)

El mar, 30 nov 2021 a las 23:01, Andresfga ***@***.***>)
escribió:

> Buenas noches, tengo un problema al solucionar esta tarea ya que me piden
> los indices de los price y de los margin que cumplan la condición dada en
> en enunciado, en si con el código calculo lo que necesito con la funcion
> index, pero la lista que me arroja contiene al final las palabras ,
> dtype='int64', name='itemid' y la parte inicial Int64Index y no se como
> solucionar esto para que solo me salga el valor del indice, ojala puedan
> ayudarme, muchas gracias.
>
> [image: Captura]
> <https://user-images.githubusercontent.com/95330160/144169638-2decbd99-e91e-4aaa-85bd-0d69669e443b.PNG>
> [image: Captura2]
> <https://user-images.githubusercontent.com/95330160/144169709-8b0da56e-7fe9-40f2-a55c-3b0626258fb9.PNG>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rramosp/ai4eng.v1/discussions/74>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADJRZIPKGCYNDZ2E5VS4HJLUOWM2HANCNFSM5JDOEKXA>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

-- 
*Aviso legal:* El contenido de este mensaje y los archivos adjuntos son 
confidenciales y de uso exclusivo de la Universidad Nacional de Colombia. 
Se encuentran dirigidos sólo para el uso del destinatario al cual van 
enviados. La reproducción, lectura y/o copia se encuentran prohibidas a 
cualquier persona diferente a este y puede ser ilegal. Si usted lo ha 
recibido por error, infórmenos y elimínelo de su correo. Los Datos 
Personales serán tratados conforme a la Ley 1581 de 2012 y a nuestra 
Política de Datos Personales que podrá consultar en la página web 
www.unal.edu.co <http://www.unal.edu.co/>.* *Las opiniones, informaciones, 
conclusiones y cualquier otro tipo de dato contenido en este correo 
electrónico, no relacionados con la actividad de la Universidad Nacional de 
Colombia, se entenderá como personales y de ninguna manera son avaladas por 
la Universidad.
